### PR TITLE
Missing missing hamburger icon label and assorted a11y issues

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary

- [X] Missing label for the hamburger icon (to open the sidebar)

- [X] Alternative test for Kolibri logo in the sidebar

- [X] Hide appbar horizontal scrolling buttons from screenreaders

- [X] Make the `KNavbarLink` components container unfocusable in Firefox

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by radinamatic** [comment]
I now realized that this introduces a new string in need of localization, so maybe targeting to `0.12.y` is not a good idea...? @indirectlylit @rtibbles

**Comment by rtibbles** [comment]
`0.12.y` is going to break string freeze, hence its longer running time frame, so this is fine.

File: kolibri/core/assets/src/views/CoreLogo/index.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
 
-  <img src="./kolibri-logo.svg">
+  <img src="./kolibri-logo.svg" alt="Kolibri logo">
```
Comment (rtibbles) [inline kolibri/core/assets/src/views/CoreLogo/index.vue]: I think we should translate this, no?

**Review by rtibbles — CHANGES_REQUESTED** [review]
Updated the failing snapshot tests - I think the alt text for the Kolibri logo should probably be translated though, so one small change, please!

File: kolibri/core/assets/src/views/CoreLogo/index.vue (hunk 1 above)
Comment (indirectlylit) [reply kolibri/core/assets/src/views/CoreLogo/index.vue]: or maybe `alt=""` if we consider the logo to be decorative?

* https://www.w3.org/WAI/tutorials/images/decorative/
* https://webaim.org/techniques/alttext/#logos

File: kolibri/core/assets/src/views/AppBar.vue [hunk 2]
```diff
@@ -123,6 +124,7 @@
     },
     mixins: [navComponentsMixin, themeMixin],
     $trs: {
+      openNav: 'Open sidebar navigation',
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/AppBar.vue]: Maybe "Open site navigation", since for screen readers 'sidebar' is less meaningful?

**Review by indirectlylit — COMMENTED** [review]
@radinamatic I pushed a merge commit to resolve the conflict

**Review by rtibbles — APPROVED** [review]

File: kolibri/core/assets/src/views/CoreLogo/index.vue (hunk 1 above)
Comment (radinamatic) [reply kolibri/core/assets/src/views/CoreLogo/index.vue]: Thanks for fixing this, such a :woman_facepalming:, forgetting to localize strings in this PR...

File: kolibri/core/assets/src/views/AppBar.vue (hunk 2 above)
Comment (radinamatic) [reply kolibri/core/assets/src/views/AppBar.vue]: It was an attempt to distinguish the sidebar menu (left), from the user menu (right).

**Pull request merged**